poc/frost.sage

```diff
@@ -247,13 +248,14 @@ class SignerParticipant(object):
         return sig_share
 
     # https://cfrg.github.io/draft-irtf-cfrg-frost/draft-irtf-cfrg-frost.html#name-aggregate
-    def aggregate(self, sig_shares, public_key_shares, commitment_list, msg):
-        binding_factors, _ = compute_binding_factors(self.G, self.H, commitment_list, msg)
+    def aggregate(self, commitment_list, msg, group_public_key, sig_shares):
+        binding_factors, _ = compute_binding_factors(self.G, self.H, group_public_key, commitment_list, msg)
         group_comm = compute_group_commitment(self.G, commitment_list, binding_factors)
 
-        participant_list = participants_from_commitment_list(commitment_list)
-        for identifier in participant_list:
-            assert(verify_signature_share(self.G, self.H, identifier, public_key_shares[identifier], sig_shares[identifier], commitment_list, self.pk, msg))
+        # does not match spec any more
+        # participant_list = participants_from_commitment_list(commitment_list)
+        #for identifier in participant_list:
+        #    assert(verify_signature_share(self.G, self.H, identifier, public_key_shares[identifier], sig_shares[identifier], commitment_list, self.pk, msg))
```

Comment:
@dconnolly I don't think we can merge this if this check fails =( Do we know what the issue is? If not, I'll investigate.